README.md

```diff
@@ -85,6 +85,44 @@ When in doubt, ask a [core team member](#core-team)! You can mention us in any i
 4.  Push to the branch: `git push origin feature/that-new-feature`
 5.  [Create a pull request](#create-a-pull-request) for your branch 🎉
 
+### Maintaining Your Fork
+Now that you have a copy of your fork, there is work you will need to do to keep it current.
+
+#### Setup Your Upstream
+Inside your dev.to directory, add a remote to the official dev.to repo:
+```
+$ git remote add upstream https://github.com/thepracticaldev/dev.to.git
+```
+
+Optionally, you can run two commands to setup and sync your fork:
+  * `npm run upstream:setup` Will perform the above command
+  * `npm run upstream:sync` Will checkout the master branch and pull the master branch from Dev.to into your local master branch.
+
+#### Rebasing from Upstream
```

Comment:
What about people who don't want to rebase?
GitHub's guide might also be worth linking: https://help.github.com/articles/syncing-a-fork/

Reply:
`npm run upstream:sync` won't be a rebase, it would be a merge.  I've included the documentation from another PR to supplement.